src/app/dashboard-widgets/applications-widget/applications-widget.component.html

```diff
@@ -1,41 +1,58 @@
 <f8-feature-toggle featureName="ApplicationsCard">
   <div class="applications-widget card-pf f8-card" user-level>
     <div class="card-pf-heading f8-card-heading">
+      <div class="card-pf-heading-details"
+           *ngIf="applicationsInProgress > 0 && (stageConfigsAvailable || runConfigsAvailable)">
+        <p>
+          Setting up {{applicationsInProgress}} applications.
```

Comment:
State needs to be checked here for `1` to be `application` instead of `applications`. E.g. `1 application`, not `1 applications`.